Cargo.lock

```diff
@@ -4,18 +4,18 @@ version = 3
 
 [[package]]
 name = "addr2line"
-version = "0.21.0"
+version = "0.24.2"
```

Comment:
It's surprising so many versions are changing. Is that required by this change?

Reply:
Not sure why they got included.

Reply:
There's a huge number of deps changing. When I reverted the dep changes locally, it seemed none of them were required by this change. I pushed the revert in eb87e6be6865b0663ab61b575d74ad815865d38f.